### ISSUE TITLE
Popup-Modal not closing on Safari/Windows

### ISSUE THREAD

**Issue by taschert** [magento/magento2#8761]
When i close a popup on Safari the overlay stay open.

### Preconditions
Windows 10
Safari 5.1.7 
Magento 2.1

### Steps to reproduce
1. Add an item to cart
2. Open the mini-cart
3. Click on remove the item from the cart
4. Validate or cancel the action

### Expected result
1. The popup should close and the grey overlay disappear

### Actual result
1. The popup is close but i still have the grey overlay over all the webite
2. The only way to remove the overlay is to refresh the page or edit html

The issue is in \vendor\magento\module-ui\view\base\web\js\modal\modal.js

In this file, in the function the function that._close(); is never called.

```
closeModal: function () {
            var that = this;

            this._removeKeyListener();
            this.options.isOpen = false;
            this.modal.one(this.options.transitionEvent, function () {
                that._close();
            });
            this.modal.removeClass(this.options.modalVisibleClass);

            if (!this.options.transitionEvent) {
                that._close();
            }

            return this.element;
        },
```

**Comment by ziiljuks** [comment]
I had same problem, I commented out from _create() function:

```
        _create: function () {
            _.bindAll(
                this,
                'keyEventSwitcher',
                '_tabSwitcher',
                'closeModal'
            );

            this.options.transitionEvent = transitionEvent;
            this._createWrapper();
            this._renderModal();
            this._createButtons();
// comment out/delete this
            //$('body').on('click', this.options.trigger,_.bind(this.toggleModal, this));
            this._on(this.modal.find(this.options.modalCloseBtn), {
                'click': this.options.modalCloseBtnHandler ? this.options.modalCloseBtnHandler : this.closeModal
            });
            this._on(this.element, {
                'openModal': this.openModal,
                'closeModal': this.closeModal
            });
            this.options.autoOpen ? this.openModal() : false;
        },

```

**Comment by magento-team** [comment]
Internal ticket to track issue progress: MAGETWO-67320

**Comment by magento-engcom-team** [comment]
@taschert thank you for your report.
The fix for this issue is already available in release 2.2.0

**Comment by nntoan** [comment]
@magento-engcom-team Can you please link the commit into this ticket?
